fbw-a380x/src/base/flybywire-aircraft-a380-842/html_ui/Pages/VCockpit/Instruments/LegacyA380X/FCU/legacy/A32NX_BaseAirliners.js

```diff
@@ -0,0 +1,1260 @@
+class BaseAirliners extends NavSystem {
+    constructor() {
+        super(...arguments);
+        this.isMachActive = false;
+        this.machTransition = 0;
+    }
+    connectedCallback() {
+        super.connectedCallback();
+        this.preserveAspectRatio("Mainframe");
+        this.addEventAlias("FMS_Upper_INC", "NavigationSmallInc");
+        this.addEventAlias("FMS_Upper_DEC", "NavigationSmallDec");
+        this.addEventAlias("FMS_Lower_INC", "NavigationLargeInc");
+        this.addEventAlias("FMS_Lower_DEC", "NavigationLargeDec");
+        this.addEventAlias("FMS_Upper_PUSH", "NavigationPush");
+        this.radioNav = new RadioNav();
+        this.radioNav.init(NavMode.FOUR_SLOTS);
```

Comment:
```suggestion
```

Reply:
yeah, also stumbled over that :) thanks!